src/_incydr_sdk/core/client.py

```diff
@@ -43,7 +43,7 @@ class Client:
     Usage example:
 
         >>> import incydr
-        >>> client = incydr.Client(url="https://api.us.code42.com", api_client_id="<client_id>", api_client_secret="<client_secret>")
+        >>> client = incydr.Client(url="https://example.com", api_client_id="<client_id>", api_client_secret="<client_secret>")
```

Comment:
very optional: since we are already following this kinda substitution format for `<client_id>` and `<client_secret`> we could also replace the example url with `<api_domain>` and make it mega clear that all 3 need to be substituted. Otherwise, LGTM!